src/NovelRenderingService.h

```diff
@@ -49,6 +50,7 @@ class NovelRenderingService {
 
   bool initializeRenderPipeline(int displayNumber);
 
+  NovelLoggingService _console;
```

Comment:
I'd prefer it if this was called `_logger` instead.

Reply:
Changed to keep it consistent throughout solution.